### DISCUSSION TITLE
PR 템플릿 (with code review)

### DISCUSSION THREAD

**Discussion by IGhost-P** [EasyAndBeauty/DailyReceipt-Flutter#5] [Ideas]
# 배경
일관된 PR과 효과적인 코드 리뷰를 위해서 템플릿을 만들었습니다.
출저는 '[요즘 우아한 개발](https://goldenrabbit.co.kr/2023/11/06/%EC%9A%94%EC%A6%98-%EC%9A%B0%EC%95%84%ED%95%9C-%EA%B0%9C%EB%B0%9C-%EA%B3%B5%ED%86%B5%EC%8B%9C%EC%8A%A4%ED%85%9C%EA%B0%9C%EB%B0%9C%ED%8C%80%EC%9D%98-%EC%BD%94%EB%93%9C-%EB%A6%AC%EB%B7%B0-%EB%AC%B8/)'에 나오는 PR 템플릿을 참고했습니다. 

# 제안 내용
### 완성된 MR 템플릿 

```
#해결하려는 문제가 무엇인가요?

#어떻게 해결했나요?

##Attachment

* 이번 MR의 Front 동작을 이해를 돕는 GIF 파일 첨부!

* 리뷰어의 이해를 돕기 위한 모듈/클래스 설계에 대한 Diagram 포함!

```
 

### 작성 예시
```
#해결하려는 문제가 무엇인가요?

* TS2305: Module “react-router” has no exported member ‘useHistory’. 오류를 내면서 빌드가 깨집니다. 다른 모듈에 의해 react-router 버전이 5 → 6으로 올라간 게 문제입니다.

# 어떻게 해결했나요?

* 사용하는 react-router의 버전을 package.json에 명시합니다.
```

**Comment by jayjeong8** [comment]
제안 감사합니다 템플릿 좋아요!! 
공유해준 글에 "리뷰어가 적은 노력으로 코드 리뷰를 잘할 수 있는 환경"을 만드는 것에 크게 공감이 가네요~! 

`##Attachment`가 이슈 템플릿의 '추가 내용'과 같은 항목이라면 용어를 통일하면 더 좋을 것 같아요!!

<img width="733" alt="image" src="https://github.com/EasyAndBeauty/DailyReceipt-Flutter/assets/82160479/8c58d851-fd71-4244-89ae-149587b6437b">

**Reply by IGhost-P** [reply]
역시 똑똑이..

**Reply by jayjeong8** [reply]
#7 에 작성해주신 PR 템플릿 적용해봤어요! 
실제로 적용해보니 h1(#) 크기가 생각보다 커보여서 h2(##)로 적용해보았습니다!  

다른 의견 있다면 편하게 알려줘요!

**Comment by gygy7151** [comment]
오옹 우아한 형제들의 PR 템플릿을 이렇게 볼
수 있어서 좋은것 같습니다!!

개인적으론 제이가 추천해준 템플릿이 이슈 트래킹 하고 원인 파악하는데 백그라운드 배경을 모르더라도 좀 더 수월 할 수 있다고 느끼는데요, 댄이 제시한 템플릿은 제가 실재로 현업에서 매 이슈마다 작성하는 템플릿이라 PR 커밋 템플릿으로 활용하면 좋지 않을까 의견 올려봅니다!!

**Reply by IGhost-P** [reply]
그렇다면

이슈 관련 (피쳐, 버그 등등 ) => 제이
PR => 댄

으로 동의하시는건가용
